packages/contracts-bedrock/src/governance/GovernanceTokenInterop.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { StorageSlot } from "@openzeppelin/contracts/utils/StorageSlot.sol";
+import { ERC20Votes } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IGovernanceDelegation } from "src/governance/IGovernanceDelegation.sol";
+import { GovernanceToken } from "src/governance/GovernanceToken.sol";
+
+contract GovernanceTokenInterop is GovernanceToken {
+    /// @notice Migrates a delegator to the `GovernanceDelegation` contract.
+    /// @param _delegator The account to migrate.
+    function migrate(address _delegator) external {
+        address currentDelegatee = delegates(_delegator);
+        if (currentDelegatee != address(0)) {
+            // `_delegates` is private, so manually clear the value in the slot (7)
+            StorageSlot.getAddressSlot(keccak256(abi.encode(_delegator, 7))).value = address(0);
```

Comment:
We are generally aligned on reducing number of dependencies so we shouldn't introduce more libraries from oz when possible. We have a storage library we already use [here](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/libraries/Storage.sol)